Lib/configparser.py

```diff
@@ -900,7 +900,14 @@ def set(self, section, option, value=None):
                 raise NoSectionError(section) from None
         sectdict[self.optionxform(option)] = value
 
-    def write(self, fp, space_around_delimiters=True):
+    def write(self, file, space_around_delimiters=True):
```

Comment:
Unfortunately, this is an API change that will break backwards compatibility (for example, `cp.write(fp=...)` won't work.) I would rather keep the old name.